tests/test_tools_pdf2txt.py

```diff
@@ -71,6 +71,9 @@ def test_contrib_hash_two_complement(self):
         """
         run('contrib/issue-00352-hash-twos-complement.pdf')
 
+    def test_contrib_excel(self):
+        run('contrib/excel.pdf', '-t xml')
+
```

Comment:
This only checks if the PDF can be parsed without errors. We have enough of those tests. You should write a unittest for `PDFLayoutAnalyzer.paint_path()`.